common-api/src/main/java/org/pmiops/workbench/config/WorkbenchConfig.java

```diff
@@ -191,9 +193,17 @@ public static WorkbenchConfig createEmptyConfig() {
     // Flag to indicate whether to enable the new Create Account flow
     // https://precisionmedicineinitiative.atlassian.net/browse/RW-3284
     public boolean enableNewAccountCreation;
+    // Flag to indicate if USER/WORSPACE data is exported to RDR
+    public boolean enableRDRExport;
   }
 
   public static class ActionAuditConfig {
     public String logName;
   }
+
+  // Defines the rdr host per environment
```

Comment:
Please expand on the comments on these fields – at a minimum it's helpful to provide an example value and some sort of pointer to who "owns" these values or where else in our system they're configured.